Tasks/Operation/01_ManageEvents.md

```diff
@@ -0,0 +1,116 @@
+
+You can create events manually in the *Tasks* module. However, most events are automatically created by the system when installing a plugin or module, such as the *Actindo Workflow Process Execution Failed*. 
+
+When the *Process Orchestration* (also known as *Workflows*) module is installed, the *Actindo Workflow Process Execution Failed* event is automatically created in the *Tasks* module. This event creates a task as soon as an error occurs in a process action. The workflow process in the *Process Orchestration* module is then paused until the specified task is performed manually by the responsible user. For detailed information about the workflow processes, see the [Process Orchestration](../../ActindoWorkFlow/Overview/01_General.md) documentation. 
+
+You can create an event manually, edit an existing event, and delete an event that is no longer needed.
+
+
+
+## Create an event
+
+You can create an event to define a scenario where a task must be created.
+
+#### Prerequisites
+
+No prerequisites to fulfill.
+
+#### Procedure
+
+*Tasks > Events*
+
+![List of events](../../Assets/Screenshots/Tasks/Events/ListEvents.png "[List of events]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Create event* view is displayed.
+
+    ![Create event](../../Assets/Screenshots/Tasks/Events/CreateEvent.png "[Create event]")
+
+2. Enter a name for the event in the *Name* field.
+
+    > [Info] If you want to provide the event name in different languages in the system, you can select the desired system language in the *"Language"* drop-down list on top of this view. By default, the following options are currently available: **English (United States)** and **Deutsch (Deutschland)**.
+
+3. Enter an identifier for the event in the *Identifier* field. The event identifier works like an attribute key. A key is required for API access and must be system-wide unique. Just like a key, the identifier must fulfill the following criteria:
+    - valid characters are **a-z** (upper and lower case), **0-9** and the underscore ( **_** )
+    - the key must not start with a number
+    - a double underscore ( **___** ) and a trailing underscore are forbidden
+
+4. Click the *Attribute set for tasks* drop-down list and select the desired option. All available attribute sets for tasks are displayed in the list.
+
+5. As you are creating the event manually, the *Plugin* drop-down list is locked.
+
+    > [Info] After saving the event, the value **Manually created** is displayed in the *Plugin* column.
+
+6. Click the *Responsible user* drop-down list and select the user to whom you want to assign the event, and the related tasks, by default. All users that are registered in the system instance are displayed in the list. 
```

Comment:
Kann ich mich hier entscheiden oder muss in jedem Fall ein user eingetragen werden? Oder kann ich alternativ eine User group angeben?

System instance: Sollten wir mal drüber sprechen. Ich habe immer current instance geschrieben.